src/clang.rs

```diff
@@ -717,6 +717,19 @@ impl Cursor {
             })
             .collect()
     }
+
+    /// Obtain the real path name of a cursor of InclusionDirective kind.
+    ///
+    /// Returns None if the cursor does not include a file, or if no real path name can be obtained
+    /// for the file.
+    pub fn get_included_file_name(&self) -> Option<String> {
+        let file = unsafe { clang_sys::clang_getIncludedFile(self.x) };
+        if file.is_null() {
+            None
+        } else {
+            Some(unsafe { cxstring_into_string(clang_sys::clang_File_tryGetRealPathName(file)) })
```

Comment:
Yeah, this is not going to be great. Can we do something like returning an struct with `{ file_name: String, path: Option<String> }`, and only compute the real path if available, using `clang_File_tryGetRealPathName::is_loaded()`?

There are a bunch of examples in this file of checking `::is_loaded()` and falling back to a reasonable behavior otherwise.